react/index.tsx

```diff
@@ -1,16 +1,77 @@
 import { canUseDOM } from 'vtex.render-runtime'
+import { PixelMessage } from './typings/events'
+import { getCategories, createIframeTag } from './helpers/index'
 
-export function handleEvents(e: PixelMessage) {
-  switch (e.data.eventName) {
-    case 'vtex:pageView': {
+const {
+  rtbhouseId,
+  location: { pathname },
+} = window
```

Comment:
This will break SSR, since it doesnt have `window` on the server. I suggest you to move this to handleMessage if possible or wrap it inside a `if (canUseDOM)`

Reply:
Resolved @klzns